src/plugins/intel_cpu/src/emitters/plugin/x64/jit_load_store_emitters.cpp

```diff
@@ -61,11 +61,16 @@ size_t store_emitter_params::hash() const {
     return seed;
 }
 
-static int get_aux_regs_as_temp(const size_t byte_size, const bool is_fill = false) {
-    if (!one_of(byte_size, 64u, 32u, 16u))
-        return 1;
+static int get_aux_regs_as_temp(const int elem_count, const int data_size,
+                                const int avx512_threshold_for_mask = 0, const bool is_fill = false) {
     if (mayiuse(cpu::x64::avx512_core) && is_fill)
         return 1;
+    if (one_of(elem_count, 4, 8, 16))
```

Comment:
It seems like safer to use:
```suggestion
    if (one_of(byte_size, 16, 32, 64))
```
The idea is only full size of xmm/ymm/zmm never use aux gpr. It's 16/32/64 bytes.
If we load 8 bytes(elem_count is 8) as bytes in vmm(no precision conversion). In the future, avx512_threshold_for_mask is adjusted to smaller than 8, we still need 1 aux gpr. what do you think?

Reply:
Discussed offline: `one_of(byte_size, 16, 32, 64)` should be checked in cases with the same precisions and `one_of(elem_count, 4, 8, 16)` - for different precisions.
Added changes. Could you take look again please?
Thanks a lot for the ideas!